extension/framework/framework.go

```diff
@@ -87,7 +87,11 @@ func TestExtensions(context *cli.Context) {
 	}
 	if runSoExt {
 		testFiles := getTestFiles(context.Args(), "so")
-		if err := gorunner.NewTestRunner(testFiles, context.Bool("verbose") || config != nil, context.String("run-test"), context.Int("parallel")).Run(); err != nil {
+		printAllLogs := context.Bool("verbose") || config != nil
+		testFilter := context.String("run-test")
+		workers := context.Int("parallel")
+		schemasFilePath := context.String("schemas-file")
+		if err := gorunner.NewTestRunner(testFiles, printAllLogs, testFilter, workers, schemasFilePath).Run(); err != nil {
```

Comment:
Shouldn't all those string literals be named constants?

Reply:
done